### ISSUE TITLE
The footer of each page should show a Last Updated date

### ISSUE THREAD

**Issue by tsimonq2** [lubuntu-team/lubuntu-manual#17]
On only the HTML view from Sphinx, the last modification date should be extracted from the Git logs and put in the footer of every page in the manual.

The dates should also be different for translations.

**Comment by ianorlin** [comment]
Good idea will probably be easier to update that way if anything changes.
